CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * CHANGE: Async Bleak adapter as default adapter on all platforms
 * CHANGE: Support only Python 3.9 and above
 * FIX: Check if async adapter in use in RuuviTagReactive
+* FIX: Fix a typo in function argument name
```

Comment:
This could same as edited PR name. Then it would be clear from changelog what has been renamed.
```
fix: corrected typo in argument name search_duration_sec
```